### PR TITLE
Fix PersonalAccountDetails header icon

### PR DESCRIPTION
**Pull request by desfero**

### CODE REVIEW

File: app/components/settings/personal-account-details/PersonalAccountDetails.tsx [hunk 1]
```diff
@@ -63,14 +62,7 @@ class AccountDetailsComponent extends React.Component<IStateProps, IOwnState> {
...
         headerText={<FormattedMessage id="settings.account-details.title" />}
-        rightComponent={
-          <ResponsiveImage
-            theme="transparent"
-            srcSet={{ "1x": personImage }}
-            alt="person image"
-            className={styles.image}
-          />
-        }
+        rightComponent={<img src={personImage} className={styles.image} alt="" />}
```
Comment (drakkein) [inline app/components/settings/personal-account-details/PersonalAccountDetails.tsx]: can we not leave empty alt?

Comment (desfero) [reply app/components/settings/personal-account-details/PersonalAccountDetails.tsx]: why do you want alt here? It's a representational icon that doesn't have any meaning (meaning was already provided by heading text that is on the right side). 'alt=""` should be used on 90% cases as often there is some more meaning text so there is no reason to bloat screen reader speech with `Account details info person image image` IMO. What do you think?

Comment (drakkein) [reply app/components/settings/personal-account-details/PersonalAccountDetails.tsx]: To remove it completely or is it required by linting?

Comment (desfero) [reply app/components/settings/personal-account-details/PersonalAccountDetails.tsx]: See https://davidwalsh.name/accessibility-tip-empty-alt-attributes. `img` without `alt` is not valid.

**Pull request merged**